www/src/pages/en/usage/first-steps.md

```diff
@@ -23,7 +23,8 @@ Of course, if you prefer to use a different auth provider, you can also use one
 5. Click "Reset Secret", copy the new secret, and add it to your `.env` as `DISCORD_CLIENT_SECRET`.
 6. Click "Add Redirect" and type in `http://localhost:3000/api/auth/callback/discord`.
    - For production deployment, follow the previous steps to create another Discord Application, but this time replace `http://localhost:3000` with the URL that you are deploying to.
-7. Save Changes. You should now be able to log in.
+7. Save Changes.
+8. Set the NEXTAUTH_SECRET as some random string too. You should now be able to log in.
```

Comment:
Maybe merge with Step 6, alongside `production deployment`, since it is [not required](https://next-auth.js.org/configuration/options#secret) in dev but will only throw an error in production.

Reply:
Can you quickly try it out by creating a new project? I just did and I can not log in without setting it to something. I am doing this on `dev`. So it doesn't just throw an error for me.

Step 6) is in the UI of discord. You can't set the `NEXTAUTH_SECRET` there. So I recommend keeping it separate.

Reply:
I think it's cause we have the secret set to empty string by default?

If it's undefined, Next-Auth will override it and set it to some hash based on your config.

Maybe the solution here is to either predefine it to something like `supersecret` or comment it out, alternatively add a `.min(1)` to the Zod schema?

Reply:
I see. How about adding a comment in `.env`, like 'Make sure not to keep the secret empty', alongside `.min(1)` in the Zod schema?

Looks like they will probably make defining the secret mandatory in the future.

> We will likely make this option mandatory, even in development, in the future.

https://next-auth.js.org/configuration/options#description-1

Reply:
> I think it's cause we have the secret set to empty string by default?
> 
> If it's undefined, Next-Auth will override it and set it to some hash based on your config.
> 
> Maybe the solution here is to either predefine it to something like `supersecret` or comment it out, alternatively add a `.min(1)` to the Zod schema?

definitely agree with `min(1)` in schema

i don't love adding something like `NEXTAUTH_SECRET=changeme` to `.env` but it's probably better than having `pnpm dev` fail out of the box

Reply:
It seems to me like we reached a consensus! :)

Added the `min(1)` check to the schema --> users will now have a meaningful error message guide them when no auth env var is set 👍 

Also added the suggested copy change to the readme. I think it reads very well with that change, no need to add a dummy string to the config. 🖖 

Anything else, guys?

Reply:
Ah okay, adding the `min(1)` breaks the build step, I get what you mean now.

Added a dummy string `changeme`.

Reply:
just noticed I changed it to `'changeme'` ... 🤦 

centering a div I mean changing a string is hard, okay?!
plus it's microsoft's fault, because code spaces

fixed it 🤖